### PR TITLE
Fix Restart Command & Spotify Credentials

### PR DESCRIPTION
**Pull request by DestinyofYeet**
After creating your pull request, tick these boxes if they are applicable to you.

- [X] I have tested my changes against the `review` branch (the latest developmental version), and this pull request is targeting that branch as a base
- [X] I have tested my changes on Python 3.8 or higher
- [X] I have ensured my code is formatted using [Black](https://github.com/psf/black)

----

### Description
Fixes the restart command

### Related issues (if applicable)

#2321

### CODE REVIEW

File: musicbot/bot.py [hunk 1]
```diff
@@ -1689,7 +1689,7 @@ async def _cmd_play(
     ):
         player = _player if _player else None
 
-        if permissions.summonplay and not player:
+        if permissions.summonplay:
```
Comment (BabyBoySnow) [inline musicbot/bot.py]: If we don't have `and not player` the constant summon messages when using the play command occur.

Comment (DestinyofYeet) [reply musicbot/bot.py]: Ah ok, not sure how it disapeared but I'll add it back in.

File: musicbot/bot.py [hunk 2]
```diff
@@ -133,6 +133,7 @@ async def _doBotInit(self):
                     aiosession=self.session,
                     loop=self.loop,
                 )
+                await self.spotify.get_token()
```
Comment (DestinyofYeet) [inline musicbot/bot.py:136]: This was missing, so it would just not even try to get the token, then complain it didn't have one

Comment (BabyBoySnow) [reply musicbot/bot.py:136]: Good catch!

**Comment by BabyBoySnow** [comment]
Works fine when restarting with guest mode, however when restarting with ID and secret throw's an error.

**Pull request merged**